user-docs/latest/getting-started/introduction.md

```diff
@@ -1,3 +1,65 @@
-# Welcome to the Strapi user documentation!
+# Welcome to the Strapi user guide!
 
-**Strapi is the open-source [Headless CMS](https://strapi.io) developers love.**
\ No newline at end of file
+This user guide contains the functional documentation related to all features available in the admin panel of your Strapi application. 
+
+![Homepage of the Admin Panel](../assets/getting-started/admin-panel-homepage.png)
+
+Before going any further into this user guide, we recommend you to acknowledge the main concepts below. They will help you to understand how Strapi works, and ensure a smooth Strapi experience.
+
+- **Development, Staging or Production Environment** <br> When starting working on your application, it is in a development environment, which is the status for the content structuration and application configuration phase. After deploying your application, it is in production or staging. This status change impacts how you can use your Strapi application, as some features are only available in development environment, such as the Content-Types Builder. In the user guide, the availability or not of a feature, depending on the application status, is always mentioned in the feature's introduction.
```

Comment:
Sorry I don't know how to do the suggestions thing so I'll try to make it clear.

`When starting working on your application` => `When you start working on your application`